nixos/modules/flyingcircus/services/sysstat.nix

```diff
@@ -51,6 +51,7 @@ in {
       description = "Run system activity accounting tool on a regular basis";
       wantedBy = [ "timers.target" ];
       after = [ "sysstat.service" ];
+
```

Comment:
Adding whitespace does not really solve the problem. I assume the intention was to delete some line.